docs-src/concepts.md

```diff
@@ -168,10 +168,21 @@ resources:
 Separation and delegation of responsibility among independent actors (e.g
 between cluster ops and application developers) can result in conflicts in the
 configuration. For example, two application teams may inadvertently submit
-configuration for the same HTTP path. There are several different strategies for
-handling this:
-
-* TODO
+configuration for the same HTTP path.
+
+In most cases, guidance for conflict resolution is provided along with the
+documentation for fields that may have a conflict. If a conflict does not have a
+prescribed resolution, the following guiding principles should be applied:
+
+* Prefer not to break things that are working.
+* Drop as little traffic as possible.
+* Provide a consistent experience when conflicts occur.
+* Make it clear which path has been chosen when a conflict has been identified.
```

Comment:
For my understanding, is this about setting the appropriate status?

Reply:
Updated to clarify that this should be done by setting conditions on status wherever possible.